neqo-http3/src/connection_client.rs

```diff
@@ -676,12 +647,13 @@ impl Http3Client {
 
         self.base_handler.add_recv_stream(
             stream_id,
-            Box::new(PushStream::new(
+            Box::new(RecvMessage::new(
+                MessageType::Response,
                 stream_id,
-                push_id,
-                Rc::clone(&self.push_handler),
                 Rc::clone(&self.base_handler.qpack_decoder),
-                Priority::default(),
+                Box::new(RecvPushEvents::new(push_id, Rc::clone(&self.push_handler))),
+                None,
+                PriorityHandler::new(true, Priority::default()),
```

Comment:
Can you flag this with a TODO: it would be good to examine what priority we set push streams relative to the request.

Reply:
I will add a TODO.
(I do not bother with ServerPush that much at the moment, since I am not sure when we will turn it on in Firefox)